src/Package/MSTest.Sdk/Sdk/Runner/ClassicEngine.targets

```diff
@@ -68,10 +68,9 @@
     <PackageReference Include="Microsoft.Testing.Extensions.CrashDump" Sdk="MSTest"
                       Version="$(MicrosoftTestingExtensionsCrashDumpVersion)" VersionOverride="$(MicrosoftTestingExtensionsCrashDumpVersion)"
                       Condition=" '$(EnableMicrosoftTestingExtensionsCrashDump)' == 'true' " />
-    <!-- TODO: Uncomment once we have CC version available for MTP v2 -->
```

Comment:
The CC dependency was removed from MSTest metapackage as well. We need to add it back there.